devtools/create_project/create_project.cpp

```diff
@@ -1075,7 +1075,7 @@ const Feature s_features[] = {
 	{      "faad",        "USE_FAAD", true, false, "AAC support" },
 	{     "mpeg2",       "USE_MPEG2", true, true,  "MPEG-2 support" },
 	{    "theora",   "USE_THEORADEC", true, true,  "Theora decoding support" },
-	{  "freetype",   "USE_FREETYPE2", true, true,  "FreeType support" },
+	{  "freetype2",  "USE_FREETYPE2", true, true,  "FreeType support" },
```

Comment:
keep it aligned

Reply:
```suggestion
	{ "freetype2",   "USE_FREETYPE2", true, true,  "FreeType support" },
```

Reply:
Dangit! I thought I had fixed that already.